scipy/stats/_binomtest.py

```diff
@@ -3,7 +3,7 @@
 from scipy._lib._util import _validate_int
 from scipy.optimize import brentq
 from scipy.special import ndtri
-from ._discrete_distns import binom
+from scipy import stats
```

Comment:
This was part of an attempt to resolve circular imports. I don't think it's needed now that I've separated `_warnings_errors.py` from `_result_classes.py`, so it can be removed if we prefer to leave this alone.